packages/xo-web/src/common/intl/messages.js

```diff
@@ -725,6 +725,7 @@ const messages = {
   customizeFilters: 'Customize filters',
 
   // ----- VM actions ------
+  cantRemotelyCopy: 'Remote copy requires at least enterprise plan',
```

Comment:
"Remotely" sounds like we're triggering the copy from somewhere else. We usually use "interpool/intrapool".